lib/projfs.c

```diff
@@ -37,6 +37,7 @@
 #include <sys/syscall.h>
 #include <attr/xattr.h>
 #include <unistd.h>
+#include <stddef.h>
```

Comment:
Ah, of course. I tend to give up on ordering includes because, as you note, sometimes there's an actual ordering dependency. I've also bumped `inttypes` up above `limits`.

Reply:
Thank you!